addons/web/static/src/search/control_panel/control_panel.scss

```diff
@@ -114,7 +114,7 @@
                     }
 
                     &.btn {
-                        padding: 0.4rem;
+                        padding: $o-cp-button-sm-no-border-padding;
```

Comment:
If I use map-get($spacers, 2), it will be 0.5rem.. As this is a fix, I'm a little conservative but ok, I can try and see what it does. Note that, I think we should refactor all spacings in the mobile CP to remove all theses margins and add paddings to make it easier to click on buttons without losing space.